storage/storage_local.go

```diff
@@ -16,8 +18,15 @@ type storageLocal struct {
 
 // NewStorageLocal : Creates a new Storage module which uses the local disk.
 func NewStorageLocal() Storage {
+	path := `F:\Code\specialedge\storage\java\`
```

Comment:
As you've already pointed out, this need to be made configurable, but fine for now.

Reply:
Yeah, annoying but I'll cover that in the next piece of work.